phoenix-core/src/main/java/org/apache/phoenix/parse/CreateSchemaStatement.java

```diff
@@ -24,7 +24,7 @@
 	private final boolean ifNotExists;
 	
 	public CreateSchemaStatement(String schemaName,boolean ifNotExists) {
-		this.schemaName = null == schemaName ? SchemaUtil.SCHEMA_FOR_DEFAULT_NAMESPACE : schemaName;
+		this.schemaName = SchemaUtil.normalizeIdentifier(schemaName);
```

Comment:
@JamesRTaylor from JIRA comment
> Identifiers are case sensitive when they are double quoted in Phoenix (and SQL in general). Otherwise they are uppercased. We shouldn’t/can’t change that.

This code change will now make it adhere to SQL standards. The `SchemaUtil.normalizeIdentifier` will help handle the case sensitivity. Otherwise with the current code, if a schema name is passed with double quotes, the full name will be taken as it is for creating the schema, which is incorrect. 

Also look for comments in `MetaDataClient` as well.